scripts/prereq-check.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+check_version()
+{
+    if [ ! $# == 2 ]; then
+        echo "Unable to determine $CHECKING version! Ensure that $CHECKING is in your path and try again." && exit 1
+    fi
+
+    local version=$1 check=$2
+    local winner=$(echo -e "$version\n$check" | sed '/^$/d' | sort -nr | head -1)
+    [[ "$winner" = "$version" ]] && return 0
+    return 1
+}
+
+echo "Checking prerequisites"
+
+REQNODEVER=8.9.0
+REQNPMVER=5.6.0
+REQGOVER=1.9.2
+REQDOCKERVER=17.0
+
+CHECKING='Node'
+NODEVER=$(sed 's/v//' <<< $(node -v))
+
+##check to see if node is installed
+type node >/dev/null 2>&1 || { echo >&2 "Mattermost requires NodeJS but it doesn't appear to be installed.  Aborting."; exit 1; }
+##check to see if it is at least the supported version
+if check_version $NODEVER $REQNODEVER; then
+    echo "Node minimum requirement met. Required: $REQNODEVER, Found: $NODEVER"
+else 
+    echo "WARNING! Mattermost did not find the minimum supported version of Node installed. Required: $REQNODEVER, Found: $NODEVER"
+    echo "We highly recommend stopping installation and updating dependencies before continuing"
+    read -p "Continue? (Y/n) " -n 1 -r
```

Comment:
So stopping is highly recommended, but continuing is the default?

Reply:
I adjusted this messaging to specifically ask for Y to move on, and note that it is not recommended to do so.